MergeCalendarsTogether.gs

```diff
@@ -167,6 +182,23 @@ function isDescWrong(event) {
   return shouldNotHaveDescButDoes
 }
 
+function AttendeeSelfStatusMatches(originEvent, mergedEvent) {
+  if (!COPY_SELF_ATTENDANCE_STATUS()) return true;
+  const originStatus = originEvent.attendees?.find(a => a.self === true)?.responseStatus
+  const mergedStatus = mergedEvent.attendees?.find(a => a.self === true)?.responseStatus
+  const matches = originStatus === mergedStatus
+  if(!matches && VERBOSE_LOGGING) console.log(`DIFF FOUND IN ATTENDEE STATUS: originStatus: ${originStatus} ; mergedStatus: ${mergedStatus}`)
+  return matches;
+}
+
+function GetAttendeeSelf(originEvent, destination) {
+  if (!COPY_SELF_ATTENDANCE_STATUS()) return [];
+  const selfAttendee = originEvent.attendees?.find(a => a.self === true);
+  if (typeof selfAttendee === 'undefined') return []
+  selfAttendee.email = destination.calendarId;
```

Comment:
The one big iffy. I couldn't find another place to correctly identify the users email address from the available variables. I can't verify if this is always an email address, but the destination record _must_ be set to the matching email address or this will fail.

Reply:
I think this is fine for now! If/when this project can work with other calendar systems (Office365...), the `CalendarId` _might_ not _necessarily_ be the email address, but that's the only situation I can think of, and even that is conjecture.